lib/jekyll/tags/highlight.rb

```diff
@@ -23,7 +23,7 @@ def initialize(tag_name, markup, tokens)
 
             #{markup}
 
-            Valid syntax: highlight <lang> [linenos]
+            Valid syntax: highlight <lang> [linenos] [highlight_lines="3 4 5"] [highlight_line_class=class_name]
           MSG
```

Comment:
Could be an interesting opportunity to link out to the jekyllrb.com page for this tag so users have a direct line to a helpful webpage with more information on the matter.

Reply:
Fixed in 8a202c50a6205fc0dda441ec8a52a686aac9a987